src/interactive/InfoToggle.jsx

```diff
@@ -23,6 +27,7 @@ const InfoToggle = ({
 	label,
 	tooltipId,
 	tooltipProps,
+	tooltipTriggerProps,
```

Comment:
Not sure if adding prop objects is the best way to go forward with this because this type of configuration becomes a mess to maintain. It might be best to rethink the api of this component. `tooltipContent` could be children and we could support a `renderTrigger` prop that would take in the icon. My concern with that would be that it introduces a breaking change.

I'm thinking we should add a deprecation warning and refactor how the tooltip works for the next major release. Thoughts? @burrata @mmcgahan

Reply:
That sounds really good - I particularly like the idea of leveraging `children` for the tooltip content. If we instead implemented a component that was just a 'standard tooltip with a label", and then used _that_ as a basis for making an 'InfoToggle', we'd be in better shape.

Deprecation + refactor sounds good, unless this is used in a small number of places, in which case a hard breaking change wouldn't be too difficult to make sooner

Reply:
I'm going to create a story in jira to cleanup this API. In the meantime we can pass the composed `InfoTooltipTrigger` with its props to the `tooltipProps` object without expanding the props passed to info toggle by overriding `trigger`.

ex:
```jsx
tooltipProps={{
  align: select('align', ['left', 'right', 'center'], 'left'),
  withClose: boolean('withClose', false),
  trigger: <InfoTooltipTrigger onClick={() => null} {...otherProps} />
}}
```
cc: @burrata @mmcgahan

Reply:
I can add that!